controllers/nutanixcluster_controller.go

```diff
@@ -278,7 +278,7 @@ func (r *NutanixClusterReconciler) reconcileCategoriesDelete(rctx *nctx.ClusterC
 	klog.Infof("%s Reconciling deletion of categories for cluster %s", rctx.LogPrefix, rctx.Cluster.Name)
 	if conditions.IsTrue(rctx.NutanixCluster, infrav1.ClusterCategoryCreatedCondition) ||
 		conditions.GetReason(rctx.NutanixCluster, infrav1.ClusterCategoryCreatedCondition) == infrav1.DeletionFailed {
-		defaultCategories := GetDefaultCAPICategoryIdentifiers(rctx.Cluster.Name)
+		defaultCategories := GetDefaultCAPICategoryIdentifiers(rctx.Cluster.Name, string(rctx.Cluster.GetUID()))
```

Comment:
How will it cleanup existing cluster categories with the old format?

Reply:
we are not supporting brownfields at this time